### ISSUE TITLE
Ask user for video quality preference to download

### ISSUE THREAD

**Issue by vishnugt** [vishnugt/youtube-dl-plugin#4]
Users should be able to select the video quality.
He should also have the freedom to set up the default quality and also an option to edit his default choice

**Comment by deferred** [comment]
@vishnugt Can I take this? If the user doesn't have a default choice set, should he be prompted each time he copies youtube URL to the clipboard with the choices available from youtube-dl?

**Comment by vishnugt** [comment]
Yeah sure, you can take it.  If the user doesn't have a default choice set, then lets not pass anything to YouTube-dl, it will automatically download the highest quality available.

**Comment by vishnugt** [comment]
We should try to validate the user input and give him some examples of video quality.  
Some might be unaware of what we mean by quality and may give "720" or "1080" as input, which is wrong.  So some sort of validation should be done.

**Comment by vishnugt** [comment]
I was thinking of maybe informing the user if he inputs a non-valid format, but I guess we can skip that since youtube-dl automatically does that.  And yeah, some examples should be nice.
